lib/compress/zstd_compress.c

```diff
@@ -5091,10 +5147,10 @@ static const ZSTD_compressionParameters ZSTD_defaultCParameters[4][ZSTD_MAX_CLEV
     { 18, 16, 17,  2,  5,  2, ZSTD_greedy  },  /* level  4.*/
     { 18, 18, 18,  3,  5,  2, ZSTD_greedy  },  /* level  5.*/
     { 18, 18, 19,  3,  5,  4, ZSTD_lazy    },  /* level  6.*/
-    { 18, 18, 19,  4,  4,  4, ZSTD_lazy    },  /* level  7 */
-    { 18, 18, 19,  4,  4,  8, ZSTD_lazy2   },  /* level  8 */
-    { 18, 18, 19,  5,  4,  8, ZSTD_lazy2   },  /* level  9 */
-    { 18, 18, 19,  6,  4,  8, ZSTD_lazy2   },  /* level 10 */
+    { 18, 18, 19+1,  4,  4,  4, ZSTD_lazy    },  /* level  7 */
+    { 18, 18, 19+2,  4,  4,  8, ZSTD_lazy2   },  /* level  8 */
+    { 18, 18, 19+3,  5,  4,  8, ZSTD_lazy2   },  /* level  9 */
+    { 18, 18, 19+4,  5,  4,  8, ZSTD_lazy2   },  /* level 10 */
```

Comment:
This is a lot of extra memory. These aren't the final params right?

Reply:
Yeah, these are the old ones - the final params have these all set as 19.